fixtures/environment.rb

```diff
@@ -0,0 +1 @@
+ARGV.each { |arg| puts ENV[arg] }
```

Comment:
I've added this to generalize retrieving env variables between operating systems. Alternatively, we could have each of the tests echo both, e.g. `echo $TEST_ENV %TEST_ENV%` and assert on either of them being substituted. LMK